model/src/main/proto/feedback_reporting.proto

```diff
@@ -256,12 +272,14 @@ message Crash {
 // Information collected about the user device's systems.
 message FeedbackReportingSystemContext {
   // The package name for the specific app version on the device.
-  string package_name = 1;
+  string package_version_name = 1;
   // The package code for the specific app version on the device.
   int32 package_version_code = 2;
-  // The ISO-639 code for the device country locale.
+  // The ISO-639 code for the country locale, as determined by the system settings. This
```

Comment:
I think your second sentence here & below is just repeating itself. Can you use an actual example for both to provide clarity? I think that's actually what's missing here.

Reply:
Done, is this clearer?

Reply:
Yes--definitely. Left one new comment below.